whitepaper.md

```diff
@@ -1,33 +1,36 @@
 
+TEN is not just a technical improvement; it represents a shift in what Web3 can become. With support for autonomous AI agents, composable encrypted dApps, and robust MEV protection, TEN unlocks applications previously thought infeasible in decentralised contexts. In doing so, TEN closes the gap between Web2's functionality and Web3's promise—offering the best of both worlds, only better.
 
 # Motivation
-Public blockchain networks have experienced a period of strong growth in 2020-2021, with new use-cases for smart contracts encompassing the finance and art worlds. Decentralised Finance (DeFi) has seen enormous inflows of capital, with the top applications seeing the equivalent of $10 billion (summer 2021) of liquidity added, and this has helped push the overall capitalisation of cryptoassets above $2 trillion (summer 2021). Meanwhile, _Non-Fungible Tokens_ (NFTs) surged in value to $10 billion (autumn 2021).
+Public blockchains have evolved significantly since the explosive growth period of 2020–2021, which saw DeFi and NFTs enter mainstream consciousness. In those years, DeFi protocols attracted over $100 billion in total value locked at their peak, and NFT markets surged to over $10 billion in trading volume. While these early use-cases validated the potential of programmable value, they also exposed structural limitations in blockchain architecture—most notably, the lack of data privacy and the exploitation of transaction ordering through Maximal Extractable Value (MEV).
 
-Public blockchains rely on the entire network seeing all transactions in order to be able to validate them and secure the network. This makes them _transparency engines_. Unfortunately, this creates a front-running issue, known as _Maximal Extractable Value_ (MEV), where miners or stakers and block proposers may steal value by observing user transactions and then preempting them. For example, a miner or bot may observe a user's desire to buy an asset at market price with an automated market maker, insert their purchase ahead in the processing queue by paying a higher transaction fee, causing the price to go up for the user, and then sell their purchase at a higher price and extract a profit from the user.
+As of 2025, the blockchain space has matured. Institutions are exploring on-chain settlement, AI agents are participating in economic activity, and privacy is no longer a luxury — it is a requirement. Yet, most public blockchains still operate as transparency engines, where all data must be exposed in order to validate computation. This makes traditional blockchains unsuitable for use-cases requiring confidentiality, from private trading strategies to personal AI agents and personal identity.
```

Comment:
> This makes traditional blockchains unsuitable for use-cases requiring confidentiality, from private trading strategies to personal AI agents and personal identity.

should we lean on the "data access" stuff here? (From the data access article)

Reply:
good idea - powerful stuff. Added